shared-libs/server-checks/src/checks.js

```diff
@@ -76,60 +93,86 @@ const checkNodeName = (nodeName, membership) => {
     membership.all_nodes.includes(nodeName);
 };
 
-const couchNodeNamesMatch = (serverUrl) => {
-  const envNodeName = process.env.COUCH_NODE_NAME;
-  const membershipUri = '/_membership';
-  const membershipUrl = serverUrl + membershipUri;
+const couchNodeNamesMatch = async () => {
```

Comment:
I think we agreed to get rid of COUCH_NODE_NAME, right? Is that a future change?

Reply:
Yea, it's a separate thing on the board I believe. This work existed long before we started discussing that :)